posthog/settings.py

```diff
@@ -119,6 +119,12 @@ def get_list(text: str) -> List[str]:
     # Add hard-coded feature flags for static self-hosted releases here
     "5440-multivariate-support",
     "new-paths-ui-edge-weights",
+    "sigma-analysis",
+    "new-sessions-player",
+    "new-sessions-player-events-list",
+    "938-breakdown-by-multiple-properties",
+    "session-insight-removal",
+    "funnel-simple-mode",
```

Comment:
For future context. This experiment is still ongoing on Cloud, but results are promising. Let's get self-hosted users on this to see how performance changes.